code-of-conduct.md

```diff
@@ -1,23 +1,27 @@
-# Code of Conduct
+# Woodcrest Code of Conduct
 
-1. **Embrace Diversity**: Woodcrest is a melting pot of roleplay experiences, languages, and styles. Whether you're a seasoned RPer or new to the scene, lead by example, offering guidance without elitism.
+Welcome to Woodcrest, a community where diversity, creativity, and respect form the bedrock of our shared experience. As members of this vibrant world, it is crucial that we adhere to a code of conduct and sim rules that uphold the integrity and enjoyment of all participants. Here is our comprehensive guide to ensuring a positive, inclusive, and enriching environment for everyone:
 
-1. **Foster Openness**: Engaging in local chat helps include everyone. While private messages (IMs) have their place, especially for welcoming those less inclined to speak up, always respect privacy and consent to avoid harassment.
+1. **Embrace and Respect Diversity**: Woodcrest is a diverse mosaic of roleplay experiences and backgrounds. We champion inclusivity and encourage all members to lead by example, fostering an environment where guidance is given with respect, not elitism.
 
-1. **Honor Boundaries**: Consent is key in roleplay. Avoid forcing unwelcome scenarios on others. Public events require participation by presence, but remember to seek approval for any significant environmental impact. Keep sensitive roleplay private and use discretion in public spaces to ensure comfort for all.
+1. **Communication is Key**: Open and respectful communication, primarily through local chat, is encouraged to maintain inclusivity. While private messages have their place, consent and privacy must always be respected to prevent harassment.
 
-1. **Minimize Drama**: Approach conflicts constructively and respect the community by avoiding slander and toxicity. Strive for peaceful resolutions and maintain a positive environment.
+1. **Consent and Boundaries**: Consent is the cornerstone of interaction here. Avoid imposing scenarios on others without their agreement. Public roleplay should not infringe on community comfort, and sensitive content must be approached with discretion. We strictly prohibit excessive violence and discriminatory language, both in public and private settings.
```

Comment:
> We strictly prohibit excessive violence

We should probably define what "excessive violence" actually means somewhere.

Reply:
**Excessive Violence:** Graphic or gratuitous depictions of violence that go beyond what is necessary for the story or scene. This includes:

1. Detailed descriptions of gore, mutilation, or extreme physical harm
2. Prolonged or repetitive acts of brutality or cruelty
3. Sexualized violence or non-consensual sexual acts
4. Glorification or trivialization of real-world atrocities or hate crimes
5. Violence targeting vulnerable or marginalized groups in a discriminatory manner

Roleplay involving combat, action, or mild violence is generally acceptable, as long as it is consensual, plot-relevant, and does not cross the line into excessive or gratuitous territory. When in doubt, err on the side of caution and respect the boundaries and comfort levels of all participants.

It's essential to note that consent and clear communication are crucial in any roleplay scenario involving violence. All parties must agree to the content and intensity of the interaction, and anyone should feel free to withdraw consent at any time if they become uncomfortable.

If you encounter instances of excessive violence, please report them to the moderation team, who will review the situation and take appropriate action in accordance with our community guidelines.

Remember, the goal is to foster a supportive and inclusive environment where everyone can engage in roleplay without feeling threatened, harassed, or exposed to content that exceeds their personal boundaries.

Reply:
We should still find a place for this, given the latest revision.

Reply:
Accommodated under Additional Notes at the bottom, as it was a definition and I didn't think it warranted its own code.